src/topotoolboxr.h

```diff
@@ -3,4 +3,7 @@
 
 void wrap_has_topotoolbox(int *a);
 
+void gradient8(float *outputR,float *demR, float *cellsizeR,int *use_mpR, int* dimsR);
```

Comment:
This needs to be  called `wrap_gradient8` to match the name of the C function you have implemented in `wrap_gradient8.c`